SAMv1.tex

```diff
@@ -229,6 +229,8 @@ \subsection{The header section}
   \multicolumn{2}{|l}{\tt @RG} & Read group. Unordered multiple {\tt @RG} lines are allowed.\\\cline{2-3}
   & {\tt ID}* & Read group identifier. Each {\tt @RG} line must have a unique {\tt ID}. The value of {\tt ID}
   is used in the RG tags of alignment records. Must be unique among all read groups in header section.  Read group IDs may be modified when merging SAM files in order to handle collisions.\\\cline{2-3}
+  & {\tt BC} & Barcode sequence identifying the sample/library.
+  In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and places a hyphen (`{\tt -}') between the barcodes from the same template.\\\cline{2-3}
```

Comment:
this isn't UMI...replace unique molecular identifiers with "Sample Barcode" (capitalized appropriatly)

Reply:
This is copied from the `SAMTag.tex`, so probably it should be changed there too (out of the scope of this PR).

Changed here.

Reply:
I think we noted this before and maybe it's in one of the other pending issues to resolve.  Anyway thanks for the change.